proto/config/devcommon.proto

```diff
@@ -50,6 +50,25 @@ message Adapter {
   org.lfedge.eve.common.PhyIoType type = 1;
   string name = 2;  // Short hand name such as "com" from bundle
   EthVF eth_vf = 3; // Only relevant for PHY_IO_TYPE_PHY_IO_NET_ETH_VF
+
+  // Optionally defines the network interface order relative to other directly assigned
+  // network devices and virtual network adapters (see "interface_order" in
+  // NetworkAdapter, file "netconfig.proto").
```

Comment:
Somewhere you should document that the numbering is across the Adapter and NetworkAdapter for a given AppInstanceConfig thus each must have a unique number (when the enforce_network_interface_order is set).